generic/tests/kdump.py

```diff
@@ -229,7 +229,7 @@ def run(test, params, env):
     timeout = float(params.get("login_timeout", 240))
     crash_timeout = float(params.get("crash_timeout", 360))
     def_kernel_param_cmd = ("grubby --update-kernel=`grubby --default-kernel`"
-                            " --args=crashkernel=128M@16M")
+                            " --args=crashkernel=128M")
```

Comment:
Not sure `crashkernel=0M-2G:128M,2G-6G:256M,6G-8G:512M,8G-:768M` or `crashkernel=auto` would be better.

Reply:
@hereischen I think specifying auto would as well be fine. Issue exists with having offset, as I told in powerpc it is advised to leave offset calculation to kdump.. which will offset the reserved memory automatically.